analyzer/habitat/analysis/arguments.py

```diff
@@ -5,9 +5,10 @@ class Arguments:
     """
     Stores representations of an operation's arguments.
     """
-    def __init__(self, args, kwargs):
+    def __init__(self, args, kwargs, debug_args):
```

Comment:
Add comment for why we need `debug_args`.

Reply:
added more comments